include/workload/shape-models/operation-space.hpp

```diff
@@ -91,7 +91,6 @@ class OperationSpace
   // API always uses flattened "Operation" points and spaces.
   // Factorized points and spaces are used internally.
 
-  OperationSpace();
```

Comment:
It works for me if I do a debug build (with -O0) but breaks with -O3. I will try and track it down, but in the meantime could you please periodically keep merging against master so that our pain is reduced when we finally merge?

Reply:
Got it. I'll merge from master periodically.

Reply:
This was due to a latent problem in the master code. Fixed in 22e837366c26cd23b94cbe040efe827999d0abcf.